docs/eventing/samples/writing-event-source/README.md

```diff
@@ -24,7 +21,6 @@ You can also write your own event source using a [ContainerSource](../../../even
 is an easy way to turn any dispatcher container into an Event Source. Similarly, another option is using [SinkBinding](../../../eventing/sources/README.md#meta-sources)
```

Comment:
@abrennan89  It looks like someone removed the actual README file without cleaning up all the links here (and throughout the library?).

This continues to work because the build script strips the `/README.md` filename and extension (which is part of whats causing the confusion). BUT.. we need to pick one way, consistently apply those changes, and backport the library because we cant change the build script if we are doing two different things.